### ISSUE TITLE
motif-cli

### ISSUE THREAD

**Issue by pedrohugorm** [erichoracek/Motif#85]
Hello,

I'm really new at iOS development, so forgive me if I'm saying something noobish.

I've added motif pod to my PodFile and it installed as it should;
I need to generate my ThemeSymbols. So, after looking everywhere, I could not find motif-cli to generate it.
I'm getting: "-bash: motif: command not found"

Any thoughts?

**Comment by erichoracek** [comment]
Hi @pedrohugorm,

To install the Motif CLI, please clone this repository, open Motif.xcworkspace, and then run the "Motif CLI" target. Afterwards, one you open a new terminal window, the `motif` command should be available.

This is documented in [here](https://github.com/erichoracek/Motif#generating-theme-symbols) in the README—let me know if any steps there are unclear and I can fix it up. Thanks!

**Comment by pedrohugorm** [comment]
Hi @erichoracek , thanks for the explanation.
I cloned the repository and ran MotifCLI target as you explained.

I faced some issues along the way:
1. XCode could not create "motif" folder in /usr/local/bin/motif - I had to create this folder manually
2. XCode could not create "Frameworks" in /usr/local/Frameworks - I had to create this one manually as well.
3. Now I'm stuck with another error: 
   **error: install_name_tool: can't map file: /usr/local/bin/motif (Invalid argument)**

**Partial info from XCode**
    export XCODE_VERSION_MINOR=0730
    export XPCSERVICES_FOLDER_PATH=/XPCServices
    export YACC=yacc
    export arch=x86_64
    export variant=normal
    /bin/sh -c /Users/xcodeclub/Library/Developer/Xcode/DerivedData/Motif-fipwhghdunoapualhqsnzsajwazq/Build/Intermediates/MotifCLI.build/Debug/MotifCLI.build/Script-3A555A2B1ACD00F800726EC9.sh

I'm looking for a possible fix. If I find it, I'll post it here.
